styles.css

```diff
@@ -0,0 +1,365 @@
+  font-weight: 700;
+}
+
+.links-container ul {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.links-container {
+  display: none;
+  background-color: #3e3c3c;
+}
+
+.header {
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: 100vh;
+  background-image: url(assets/Illustration-header.png);
+  background-size: 40%;
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.header .job-desc {
+  font-family: "Roboto", sans-serif;
+  color: #fff5e1;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 32px;
+  margin-left: 10px;
+}
+
+.header .intro {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff5e1;
+  max-width: 50%;
+  margin-left: 10px;
+}
+
+.icons-container > ul {
+  display: flex;
+  justify-content: space-between;
+  max-width: 50%;
+  margin-left: 10px;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.recent-works {
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: 100vh;
+  align-items: center;
+}
+
+.recent-works h2 {
+  margin-top: 30px;
+  font-family: "Crete Round", sans-serif;
+  color: #1c1a19;
+}
+
+hr {
+  width: 85%;
+}
+
+.head-work {
+  display: flex;
+  flex-direction: column;
+  min-height: 80vh;
+}
+
+.head-work img {
+  width: 80%;
+  align-self: center;
+}
+
+.head-work h2 {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.recent-works p {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.techstack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+}
+
+.techstack-container li {
+  font-family: 'Inter', sans-serif;
+  border: 1px solid #8993a4;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  list-style-type: none;
+}
+
+.head-work .project-button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: flex-start;
+  margin-left: 36px;
+  margin-top: 20px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+}
+
+.work .project-button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: left;
+  margin-top: 22px;
+  margin-left: 1.5px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+  width: 99%;
+  border: none;
+}
+
+.head-work .project-button:hover {
+  background: rgba(0, 0, 0, 0.3);
+}
```

Comment:
- [ ] Kindly add the correct hover styles to be similar to the Figma template 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/219783675-df19d77f-264d-4f15-935a-f977643d80f4.png)